### ISSUE TITLE
Can't retrieve translation

### ISSUE THREAD

**Issue by clmt-san** [danhper/python-i18n#26]
Hi!

I'm trying to implement i18n in one of my python script. I tried to figure out how this lib works but I'm facing an issue. I didn't find how to retrieve the desired value in my yml file:

python script:

```    
    i18n.load_path.append('template')
    i18n.set('file_format', 'yaml')
    i18n.set('locale', 'en')
    print(i18n.t('template.subject'))
```

yaml file (template.en.yml):

```
en:
  subject: Test %{filename}
```

But my script only prints "template.subject" and not the string.

What am I missing here ?

Thanks!

**Comment by fabianjs** [comment]
Hi,

Not the maintainer but was facing the same issue. Switching to json worked for me (see python-i18n/i18n/tests/resources/translations/en.json for an example).

You will need to set 

i18n.set('file_format', 'json')
i18n.set('filename_format', '{locale}.{format}')

and finally of course
i18n.load_path.append('path/to/translations/folder')

for this to work. 

I would however be really interested in the correct YAML solution which I couldn't get to work either.

**Comment by clmt-san** [comment]
I actually achieved to make it work with YAML. By default, python-i18n uses YAML for external files. I just removed the i18n.set format thingy. 

Also, I struggled a little bit with the path structure (and It's not that easy to find the issue because if the code does not find the right translation, It will print you the path that you're searching and not something like "Error translation not found" ^^)

Here is my code: 

`    i18n.load_path.append('template')`
  `  i18n.set('locale', countrycode)  ` 
      ` subject = i18n.t(f'{countrycode}.mail.subject', filename=filename)  ` 
     ` body = i18n.t(f'{countrycode}.mail.body', filename=filename, error=i18n.t(f'{countrycode}.error.{errorcode}'))`

And my paths structured like this:
`template/en/mail.en.yml`
